feature-configs/demo/gatekeeper/sample-testcases.yaml

```diff
@@ -0,0 +1,49 @@
+---
+# This should fail to create due to the specific toleration
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: test-fail-specific-
```

Comment:
The pods should be in the "gatekeeper-demo" namespace due to the namespace selector on the constraint templates.